app/javascript/packages/verify-flow/steps/password-confirm/forgot-password.tsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from 'react';
+import { PageHeading, Button } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { getAssetPath } from '@18f/identity-assets';
+import { FormStepsContext } from '@18f/identity-form-steps';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const { resetPasswordUrl } = useContext(FormStepsContext);
+
+  function goToResetPassword() {
+    window.location.href = resetPasswordUrl!;
+  }
+
+  return (
+    <>
+      <img
+        src={getAssetPath('status/info-question.svg')}
+        width="50"
+        alt=""
```

Comment:
We should have a non-empty alt for this, to match other status pages.

```suggestion
        alt={t('components.status_page.icons.question')}
```